docs/pages/versions/unversioned/sdk/media-library.mdx

```diff
@@ -175,6 +175,17 @@ const styles = StyleSheet.create({
 
 </SnackInline>
 
+## Known limitations
+
+### Empty albums
+
+Due to system limitations on Android, it is not possible to create empty albums. It is necessary to either pass an existing asset to add to the album or a URI of a local resource, which will be used to create a new asset inside the album.
```

Comment:
```suggestion
Due to system limitations on Android, it is impossible to create empty albums. It is necessary to either pass an existing asset to add to the album or a URI of a local resource, which will be used to create a new asset inside the album.
```

Sorry I missed this in my previous review. Let's fix the Vale warning about wordiness.